src/app/wallets/send-on-chain.ts

```diff
@@ -291,7 +302,7 @@ const executePaymentViaIntraledger = async <
           sendAll,
           paymentAmounts: paymentFlow,
 
-          amountDisplayCurrency: converter.fromUsdAmount(paymentFlow.usdPaymentAmount),
+          amountDisplayCurrency: amountDisplayCurrencyAsNumber,
```

Comment:
`displayCurrency` needs to be updated (similar to intraledger and lightning). Also `additionalCreditMetadata` needs to be updated

Reply:
https://github.com/GaloyMoney/galoy/pull/2383#discussion_r1122315709